rpcserver.go

```diff
@@ -95,6 +95,10 @@ func (s *accountStore) PendingBatch() error {
 func newRPCServer(server *Server) *rpcServer {
 	accountStore := &accountStore{server.db}
 	lndServices := &server.lndServices.LndServices
+	batchVersion, err := server.determineBatchVersion()
+	if err != nil {
+		rpcLog.Warnf("unable to determine the batch version: %v", err)
```

Comment:
This should be a hard error, not just logging it IMO.

Reply:
fixed